pkg/tbtc/tbtc.go

```diff
@@ -23,14 +25,13 @@ func Initialize(
 	node := newNode(chain, netProvider, persistence)
 	deduplicator := newDeduplicator()
 
-	// TODO: Uncomment once https://github.com/keep-network/keep-core/pull/3094 is merged.
-	// err := sortition.MonitorPool(ctx, chain, sortition.DefaultStatusCheckTick)
-	// if err != nil {
-	// 	return fmt.Errorf(
-	// 		"could not set up sortition pool monitoring: [%v]",
-	// 		err,
-	// 	)
-	// }
+	err := sortition.MonitorPool(ctx, chain, sortition.DefaultStatusCheckTick)
```

Comment:
I haven't had a chance to test it but how does the logs from the monitoring loop look like? Are we not having the same logs duplicated for Random Beacon and TBTC? Is there any way for the client operator to distinguish which sortition pool is up to date? See [this](https://github.com/keep-network/keep-core/pull/3093#discussion_r928722939) conversation please.

Reply:
It seems the logs from monitoring Random Beacon and TBTC are indeed duplicated now, e.g.:
```
2022-07-26T11:11:01.422+0200    INFO    keep-sortition   sortition/sortition.go:65 checking sortition pool operator status
```

and then later:
```
2022-07-26T11:11:01.441+0200    INFO    keep-sortition   sortition/sortition.go:65 checking sortition pool operator status
```